jetty-ee10/jetty-ee10-proxy/src/main/java/org/eclipse/jetty/ee10/proxy/ProxyServlet.java

```diff
@@ -90,7 +90,14 @@ protected void service(HttpServletRequest request, HttpServletResponse response)
                     try
                     {
                         Request.Content content = proxyRequestContent(request, response, proxyRequest);
-                        Content.copy(content, delegate, Callback.from(delegate::close, x -> onClientRequestFailure(request, proxyRequest, response, x)));
+                        Content.copy(content, delegate, Callback.from(
+                            delegate::close,
+                            x ->
+                            {
+                                delegate.fail(x);
+                                onClientRequestFailure(request, proxyRequest, response, x);
+                            })
```

Comment:
Can the delegate throw?  Should this be:
```suggestion
                            x ->
                            {
                                ExceptionUtil.callAndThen(x, delegate::fail, x - > onClientRequestFailure(request, proxyRequest, response, x));
                            })
```